addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -246,12 +244,9 @@ def _fetch_next_step(self, selected_answer_ids):
             -> NOK
         """
         self.ensure_one()
-        domain = [('chatbot_script_id', '=', self.chatbot_script_id.id), ('sequence', '>', self.sequence)]
+        domain = Domain('chatbot_script_id', '=', self.chatbot_script_id.id) & Domain('sequence', '>', self.sequence)
```

Comment:
I always wonder if we should prefer `Domain([...])` or `Domain(...) & Domain(...)`.

Do we have plans to "deprecate" the list syntax at some point?

Because as it is, for simple cases, the `Domain([...])` is shorter to type and to read (and the more elements there is, the shorter it is).

I like to use operators when combining domains from different places, but I'm still not convinced by them when defining simple multi-part domains.

Reply:
There is no plan to deprecate the list syntax. `Domain([...])` just feels more familiar.
(see my other comment about how I chose which one to use)